src/TargetPlugin.groovy

```diff
@@ -27,6 +28,15 @@ class TargetPlugin implements TerraformPlanCommandPlugin, TerraformApplyCommandP
                .each { item -> command.withArgument("-target ${item}") }
     }
 
+    @Override
+    public void apply(TerraformDestroyCommand command) {
+        def targets = Jenkinsfile.instance.getEnv().RESOURCE_TARGETS ?: ''
+        targets.split(',')
+               .collect { item -> item.trim() }
+               .findAll { item -> item != '' }
+               .each { item -> command.withArgument("-target ${item}") }
+    }
+
     @Override
```

Comment:
Does the TargetPlugin apply in the case of destroy? If you wanted to remove a single resource without touching anything else, couldn't you use a normal pipeline, remove your resource from your code, and target just the removed resource?

If we can avoid changing plugins at all, I would love that.